src/view/LoggedInView.java

```diff
@@ -7,33 +7,59 @@
 import java.beans.PropertyChangeEvent;
 import java.beans.PropertyChangeListener;
 
+// This is a general view connecting all other main views via simple buttons.
 public class LoggedInView extends JPanel implements ActionListener, PropertyChangeListener {
 
     public final String viewName = "logged in";
     private final LoggedInViewModel loggedInViewModel;
 
+    private final ViewManagerModel viewManagerModel;
+
     JLabel username;
 
     final JButton logOut;
 
+    final JButton leaderboard;
+    final JButton createTeam;
+    final JButton compareTeams;
+    final JButton getPlayerStats;
```

Comment:
This may need to be removed and added to the create team view as discussed in our last meeting.